soc/wch/ch32v00x/Kconfig.defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_CH32V003
+
+config NUM_IRQS
+	default 48
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 48000000
+
+config MAIN_STACK_SIZE
+	default 512
+
+config IDLE_STACK_SIZE
+	default 256
+
+config ISR_STACK_SIZE
+	default 256
```

Comment:
I already worked with very low stacks. I had problems when using stacks with 256 bytes when multi-threading is enabled. Could you run applications like philosophers at least?

I had very good results when using single threading since this have very low sram size.

Reply:
The `philosophers` examples is too big for this base chip (`zephyr/zephyr_pre0.elf section `noinit' will not fit in region `RAM'`).

@nzmichaelh has an example "video game platform" working with these settings just fine.